### PR TITLE
docs: update standard for check-node-version NPM script

### PR DESCRIPTION
**Pull request by pbredenberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,14 +121,15 @@ NPM script.
 
 ### check-node-version
 
-Add a `check-node-version` task to package.json, providing the desired version of Node.js
-and NPM that you wish to enforce. Execute it as part of the `test` NPM script as well.
+Add a `check-node-version` task to package.json, providing the desired version of
+Node.js and NPM that you wish to enforce. Execute it as part of the `standards`
+NPM script as well.
```
Comment (onebytegone) [inline README.md:127]: Looking at some of the newer pipelines we've upgraded, `npm run check-node-version` is called from the CI config right after `npm ci`. `check-node-version` is being used to validate that the dependencies were installed with the version of npm that we expected. If it's in `npm run standards`, developers will run `check-node-version` when trying to lint the code. 

Could we update the "Execute it..." part to indicate `npm run check-node-version` should be in the CI definition? Open to other ideas as to how to best validate NPM versions.

File: README.md [hunk 2]
```diff
@@ -121,14 +121,15 @@ NPM script.
...
 
 ```json
 {
    "scripts": {
       "check-node-version": "check-node-version --node 16.15.0 --npm 8.5.5",
```
Comment (onebytegone) [inline README.md:131]: Do we still enforce Node versions this way? I vaguely thought we left "you're using the right version of node" to `nvm` now. Could be mistaken though 🤷‍♂️

**Review by onebytegone — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pbredenberg) [reply README.md]: Ah, I see. For a time we were locking the Node.js version in private repos, but not in public repos that support multiple versions. I didn't realize that wasn't desired.

File: README.md (hunk 1 above)
Comment (pbredenberg) [reply README.md:126]: Updated!

**Pull request merged**